actors/miner/tests/util.rs

```diff
@@ -1612,6 +1614,69 @@ impl ActorHarness {
         expirations
     }
 
+    pub fn report_consensus_fault(
```

Comment:
please wait for #348 to land and then rebase on it

Reply:
Rebased onto main now that #348 has been merged.